### ISSUE TITLE
17.0.0 - Custom process made Onfido unusable

### ISSUE THREAD

**Issue by romainguefveneu** [onfido/onfido-android-sdk#251]
#### What was the expected behaviour?

Onfido should not start another [`Application`](https://developer.android.com/reference/android/app/Application)

#### What happened instead?

Having a custom process for Onfido starts another [`Application`](https://developer.android.com/reference/android/app/Application), which is not wanted, and crashes the Firebase initialization.

#### Version info:

- `onfido-capture-sdk` version:17.0.0
- Android API version(s):All

- `compileSdkVersion`:All
- `targetSdkVersion`:All
- `minSdkVersion`:All
- Support library version (if applicable):

#### Steps to reproduce:

- Create a project with a [`Application`](https://developer.android.com/reference/android/app/Application) class
- Start Onfido 
- Another [`Application`](https://developer.android.com/reference/android/app/Application) class is created

#### Issue severity information (number of users/crashes if applicable):

Unusable

#### Other useful info:
_screenshots, videos, etc._

**Comment by jacques-sendwave** [comment]
We are experiencing the same issue here after updating to 17.0 or 18.0, Firebase is crashing our App because Onfido runs in a separate process and even if we try to manually init Firebase after starting this new process, there are other things crashing, like LeakCanary... Because this new process has to reinitialize everything from the Application.

Is there any new Onfido setup we should do or any way to opt out of this new Onfido-specific process?

**Comment by jacques-sendwave** [comment]
Hi @onfido-android-ci , I get your point but since it's a process out of our project the SDK should at least provide a way to identify that the current process is from Onfido so we can avoid trying to init/start unnecessary things. Would that be something you could include in your SDK?

**Comment by romainguefveneu** [comment]
@onfido-android-ci Can you provide a flag to disable this instead? We shouldn't have to do all of this because you want another process.

**Comment by jacques-sendwave** [comment]
Hi @onfido-android-ci , even after manually starting Firebase and adding this condition to not start other libraries when the current process is Onfiso, we still get a crash... I agree it would be better to be able to disable Onfido from running in another process.

Below is the crash we get when starting Onfido using v18.0.0:

```java.lang.RuntimeException: Unable to start activity ComponentInfo{com.mychime.waveremit.app.staging/com.onfido.android.sdk.capture.ui.OnfidoCoreActivity}: java.lang.NullPointerException

FATAL EXCEPTION: main
Process: com.mychime.waveremit.app.staging:onfido_process, PID: 28292
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.mychime.waveremit.app.staging/com.onfido.android.sdk.capture.ui.OnfidoCoreActivity}: java.lang.NullPointerException
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3449)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3601)
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:85)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2066)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:223)
	at android.app.ActivityThread.main(ActivityThread.java:7656)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
Caused by: java.lang.NullPointerException
	at com.onfido.android.sdk.capture.common.SdkController.getSdkComponent(Unknown Source:39)
	at com.onfido.android.sdk.capture.common.SdkController.getSdkComponent$default(Unknown Source:11)
	at com.onfido.android.sdk.capture.ui.OnfidoCoreActivity.onCreate(Unknown Source:11)
	at android.app.Activity.performCreate(Activity.java:7994)
	at android.app.Activity.performCreate(Activity.java:7978)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1309)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3422)
	... 11 more

**Comment by romainguefveneu** [comment]
@rocklass 👋 Could you create another issue for this? This issue is about the "Custom process".

**Comment by rocklass** [comment]
> @rocklass 👋 Could you create another issue for this? This issue is about the "Custom process".

I can but it seems strongly related.

**Comment by rocklass** [comment]
Thank you Musa, looking forward to testing this new release! 🎉

**Comment by rocklass** [comment]
Hi 👋 
I confirm release 19.0.0 solves the callback issues 💪 
Thanks!

**Comment by rocklass** [comment]
Hi Musa 👋 
It is possible to mark the issue as solved and to identify the issues solved in version 19.0.0?
Thanks!

**Comment by jacques-sendwave** [comment]
Hi @onfido-android-ci , 
Yeah, it looks like this new version fixed the issues we were facing 🎉 
Thanks!
